### PR TITLE
Alcatel-Lucent Aos7 increase fan threshold

### PR DESCRIPTION
**Pull request by paulierco**
Increase fan threshold for aos7 to avoid any false alarms

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by Jellyfrog — DISMISSED** [review]
There is no Oid for this I assume?

**Comment by paulierco** [comment]
> There is no Oid for this I assume?

No OID for fan thresholds :(

**Comment by paulierco** [comment]
Wait, new tests because of the bgp PR ?

**Comment by Jellyfrog** [comment]
Merge conflict yes

**Comment by PipoCanaja** [comment]
Hi @paulierco 
Why do we have such a huge change on tests ? I would have expected only a few lines to bu updated because of the limit change ?

**Comment by paulierco** [comment]
> 
> 
> Hi @paulierco
> Why do we have such a huge change on tests ? I would have expected only a few lines to bu updated because of the limit change ?

Let me try again. I updated them from [here](https://github.com/librenms/librenms/pull/13047), generate new tests and update the PR. Something went wrong ? Also, tests are passing.

**Comment by paulierco** [comment]
Looks fine now.

**Comment by PipoCanaja** [comment]
Much better indeed. The test file "changes" match the code "changes" :)

**Review by PipoCanaja — APPROVED** [review]
LGTM
